sdk/python/kfp/compiler/_read_write_test_config.py

```diff
@@ -46,6 +46,7 @@
             'two_step_pipeline_containerized',
             'pipeline_with_multiple_exit_handlers',
             'pipeline_with_parallelfor_parallelism',
+            'pipeline_in_pipeline',
         ],
```

Comment:
LGTM! There are few comments:
1. This might be irrevalent, but does this feature allow a component to call itself recursively? Or is it possible that the call stack is like A -> B -> C -> A?
2. Maybe also put the two compiler tests in v2 sample tests as well to make sure they pass.

Reply:
Going to try adding sample test later.